clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaStoreIngestionService.java

```diff
@@ -536,6 +543,14 @@ public boolean startInner() {
           Executors.newSingleThreadExecutor(new DaemonThreadFactory("ParticipantStoreConsumptionTask"));
       participantStoreConsumerExecutorService.submit(participantStoreConsumptionTask);
     }
+    final int idleIngestionTaskCleanupIntervalInSeconds = serverConfig.getIdleIngestionTaskCleanupIntervalInSeconds();
+    if (idleIngestionTaskCleanupIntervalInSeconds > 0) {
```

Comment:
nit: idleStoreIngestionTaskKillerExecutor != null check seems cleaner ? ie

```
if (idleStoreIngestionTaskKillerExecutor != null) {
 this.idleStoreIngestionTaskKillerExecutor.scheduleWithFixedDelay(
          this::scanAndCloseIdleConsumptionTasks,
          idleIngestionTaskCleanupIntervalInSeconds,
          idleIngestionTaskCleanupIntervalInSeconds,
          TimeUnit.SECONDS);
    }
```

Reply:
Yeah, good to be consistent with other service.